src/components/data-management/Header.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Dropdown, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+
+const { Title, Text, Paragraph } = Typography;
+
+const Header = (props) => {
```

Comment:
I would rename this component: I wouldn't call it `Header`, because we already have `Header` in Plots and Tables and Data Exploration that means something else. This seems to be about the Project buttons, it is more of a project menu kind of thing. Definitely not a Header.